spec/models/user_spec.rb

```diff
@@ -22,6 +22,14 @@
     it { should have_many(:withdrawals) }
```

Comment:
aunque no es de este PR, `should` está medio deprecado y recomiendan usar `is_expected` (https://relishapp.com/rspec/rspec-core/v/3-8/docs/subject/one-liner-syntax)